driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedServerDiscoveryAndMonitoringTest.java

```diff
@@ -38,4 +42,17 @@ public UnifiedServerDiscoveryAndMonitoringTest(@SuppressWarnings("unused") final
     public static Collection<Object[]> data() throws URISyntaxException, IOException {
         return getTestData("unified-test-format/server-discovery-and-monitoring");
     }
+
+    @Before
+    public void before() {
+        skipTests(getDefinition());
+    }
+
+    public static void skipTests(final BsonDocument definition) {
+        String description = definition.getString("description", new BsonString("")).getValue();
+        assumeFalse("Skipping because our server monitoring events behave differently for now",
+                description.equals("connect with serverMonitoringMode=auto >=4.4"));
+        assumeFalse("Skipping because our server monitoring events behave differently for now",
+                description.equals("connect with serverMonitoringMode=stream >=4.4"));
+    }
```

Comment:
This will be undone by JAVA-5230 as per https://jira.mongodb.org/browse/JAVA-5230?focusedCommentId=6118489&focusedId=6118489&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-6118489.